src/components/Home.js

```diff
@@ -1,36 +1,13 @@
 import { useState, useEffect } from "react";
-import weather from "../apis/weather";
 
 const Home = () => {
   let [userName, setUserName] = useState("");
 
-  const getWeatherData = async (latitude, longitude) => {
-    const response = await weather.get("/weather");
-    let data = response.data;
-    let dataObj = JSON.parse(data.replace("test(", "").replace(")", "").trim());
-    let temperature = dataObj.main.temp - 273.15;
-    localStorage.setItem("temperature", Math.round(temperature));
-    localStorage.setItem("data", data);
-  };
   const handleClick = () => {
     localStorage.setItem("user-name", userName);
+    window.location.reload(false);
   };
 
-  useEffect(() => {
-    if (navigator.geolocation) {
-      navigator.geolocation.getCurrentPosition(
-        (position) => {
-          let latitude, longitude;
-          latitude = position.coords.latitude;
-          longitude = position.coords.longitude;
-          getWeatherData(latitude, longitude);
-        },
-        (error) => {
-          console.log("Error: ", error);
```

Comment:
console.error would be semantically correct here

Reply:
Updated